app/presenters/callback.rb

```diff
@@ -0,0 +1,27 @@
+class Callback
+  class InvalidCallbackTimeError < StandardError; end
+
+  attr_reader :request_reference
+
+  def initialize(time_slot:, request_reference:)
+    @time_slot = time_slot
+    @request_reference = request_reference
+  end
+
+  def time
+    case @time_slot
+    when ['morning']
+      'between 8am and 12pm'
```

Comment:
The view prefixes all of these with 'between', so that isn't needed here.

Reply:
👍 
My thought was originally that this could support different formats of slot e.g. `before 5pm`, but in retrospect I think that's total YAGNI.